docs/resources/routing_skill_group.md

```diff
@@ -0,0 +1,125 @@
+---
+page_title: "genesyscloud_routing_skill_group Resource - terraform-provider-genesyscloud"
+subcategory: ""
+description: |-
+  Genesys Cloud Routing Skill Group
+---
+# genesyscloud_routing_skill_group (Resource)
+
+Genesys Cloud Routing Skill Group
+
+## API Usage
+The following Genesys Cloud APIs are used by this resource. Ensure your OAuth Client has been granted the necessary scopes and permissions to perform these operations:
+
+* [GET /api/v2/routing/skillgroups](https://developer.genesys.cloud/platform/preview-apis#get-api-v2-routing-skillgroups)
+* [POST /api/v2/routing/skillgroups](https://developer.genesys.cloud/platform/preview-apis#post-api-v2-routing-skillgroups)
+* [PATCH /api/v2/routing/skillgroups/{skillGroupId}](https://developer.genesys.cloud/platform/preview-apis#patch-api-v2-routing-skillgroups--skillGroupId-)
+* [DELETE /api/v2/routing/skillgroups/{skillGroupId}](https://developer.genesys.cloud/platform/preview-apis#delete-api-v2-routing-skillgroups--skillGroupId-)
+
+**NOTE: The Routing Skill Groups feature is currently in beta and preview.  This means while the APIs are available in preview (they can change).  The feature is in a closed beta.  You must be part of the beta in order to use this resource.  If you are not part of the beta, ths resource will not function properly for you.  Please contact Chris Bohlin (chris.bohlin@genesys.com) to be added to the beta.
```

Comment:
Looks like you've manually wrote stuff here. Usually we auto generate these files with `go generate` using the resource description, `resource.tf` and `apis.md`. If you put the right stuff into those files it should be able to pick them up and write this file.

Reply:
Oopps.  I never realized we auto-generated these files.  Any reason why we check the generated doc files into source control.  I will fix

Reply:
Fixed